### PR TITLE
Fix #694 cursor is positioned at the end of editText

### LINKED ISSUE #694: Cursor position should be at the end of the editText
**Actual Behaviour**

Currently when we open the ControlFragmentAdvanced,and select an editText the cursor is positioned at the beginning of the  editText and the user has to the move the cursor to the end and then change the value.

**Expected Behaviour**

Cursor should be present at the end of the editText

**Steps to reproduce it**

Open the app.
Select applications.
Open Control.
Select Advanced fragment
Click on any editText

**Screenshots of the issue**

screenshot_20180112-181403

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by nikit19**
Fixes issue #694 

Changes:Cursor is positioned at the end of editText

Screenshots for the change: 
ezgif com-video-to-gif 30

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java [hunk 1]
```diff
@@ -90,8 +92,10 @@ public void onClick(View v) {
                     data = data > maxima ? maxima : data;
                     data = data < minima ? minima : data;
                     editText.setText(String.valueOf(data));
+                    editText.setSelection(String.valueOf(data).length());
                 } catch (Exception e) {
                     editText.setText("0");
+                    editText.setSelection("0".length());
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java]: Could this be simply put as 1?

File: app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java [hunk 2]
```diff
@@ -122,5 +126,6 @@ public String getText() {
 
     public void setText(String text) {
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java]: I wonder what the use of this function! :thinking: in the onclick implementation it's the same lines is it?

Comment (nikit19) [reply app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java]: Actually the onClick implementation is for the button clicks. When the value gets incremented or decremented the cursor will also remain at the end.
While this setText method is for EditTextWidget so when this dialog is opened,the cursor will be seen at the end

File: app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java (hunk 1 above)
Comment (nikit19) [reply app/src/main/java/org/fossasia/pslab/others/EditTextWidget.java]: Sure :+1:

**Comment by CloudyPadmal** [comment]
Could you please check why the Travis build is failing? PR looks great! 👍

**Comment by nikit19** [comment]
@CloudyPadmal travis build has passed now

**Pull request merged**